.github/workflows/review-app-comment-trigger.yml

```diff
@@ -0,0 +1,147 @@
+name: Trigger Review App Deployment from Comments
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  process_comment:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/deploy') }}
+
+    steps:
+      - name: Checkout code to access workflow file
+        uses: actions/checkout@v4
+
+      - name: Extract available sites
+        id: available_sites
+        uses: mikefarah/yq@de2f77b49cbd40fd67031ee602245d0acc4ac482
+        with:
+          cmd: yq '.on.workflow_dispatch.inputs.site.options[]' '.github/workflows/review-app.yml'
+
+      - name: Format available sites
+        id: format_sites
+        run: |
+          # Format the sites as a comma-separated list, excluding ecospheres
+          AVAILABLE_SITES=$(echo "${{ steps.available_sites.outputs.result }}" | grep -v "ecospheres" | tr '\n' ',' | sed 's/,$//')
+          echo "AVAILABLE_SITES=$AVAILABLE_SITES" >> $GITHUB_ENV
+          echo "available_sites=$AVAILABLE_SITES" >> $GITHUB_OUTPUT
+          echo "Available sites: $AVAILABLE_SITES"
+
+      - name: Get PR details
+        id: pr_details
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const prNumber = context.issue.number;
+            core.setOutput('pr_number', prNumber);
+
+            // Get PR information to verify it's from the same repository
+            const { data: pullRequest } = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: prNumber
+            });
+
+            const isSameRepo = pullRequest.head.repo.full_name === context.repo.full_name;
+            core.setOutput('is_same_repo', isSameRepo.toString());
+
+            return { prNumber, isSameRepo };
+
+      - name: Parse deployment command
+        id: parse_command
+        if: ${{ steps.pr_details.outputs.is_same_repo == 'true' }}
+        run: |
+          COMMENT="${{ github.event.comment.body }}"
+          AVAILABLE_SITES="${{ env.AVAILABLE_SITES }}"
+
+          # Extract sites from command
+          if [[ $COMMENT =~ /deploy\ +([a-zA-Z0-9,-]+) ]]; then
```

Comment:
That comma got me wondering for a while. I'd move it up front and possibly add a comment? Or go with space-delimited site names?